e2etests/tests/session.go

```diff
@@ -178,6 +178,84 @@ var _ = ginkgo.Describe("Advertisement", func() {
 			for _, c := range frrs {
 				ValidateFRRPeeredWithNodes(nodes, c, family)
 			}
+
+		},
+			ginkgo.Entry("IPV4", ipfamily.IPv4),
+			ginkgo.Entry("IPV6", ipfamily.IPv6),
+		)
+
+		ginkgo.DescribeTable("Establishes sessions with graceful restart", func(family ipfamily.Family) {
```

Comment:
I'd merge this and the other e2e test commit

Reply:
````
* e53addc - (HEAD -> graceful-restart, karampok/graceful-restart) E2E: add integration test for Graceful Restart (3 hours ago) <karampok>
* 439ef66 - Throw warning when graceful restart flag changes (4 days ago) <karampok>
* 56e8b42 - E2E: correct ginkgo describe name for session.go file (7 days ago) <karampok>
* 9285c6c - E2E: extend tests to check graceful restart (7 days ago) <karampok>
* 6d571ed - API: add graceful restart support (7 days ago) <karampok>
````

Do you want at the end to have only two commits?

Reply:
we said it was fine not to :P